### PR TITLE
Addition of bits to aggregation header + length clarifications

### PR DESCRIPTION
**Pull request by aboba**
Fix for Issue https://github.com/AOMediaCodec/av1-rtp-spec/issues/24

### CODE REVIEW

File: av1-rtp-spec.md [hunk 1]
```diff
@@ -199,6 +199,13 @@ previous OBU, 0 otherwise
 Y: set to 1 if the last OBU contained in the packet will continue in another
 packet, 0 otherwise
 
+X: if set to 0, each OBU (or OBU fragment) MUST be preceded by a length
```
Comment (bjorn-terelius) [inline av1-rtp-spec.md]: As discussed in the last meeting, this is not sufficient. The parser won't know which fragment is the last and therefore won't know whether it should attempt to read a length field.

File: av1-rtp-spec.md [hunk 2]
```diff
@@ -216,12 +223,23 @@ The design allows combination of aggregation and fragmentation, i.e., allow for
...
 additional bytes are used in the representation (AV1, Section 4.10.5).
 
+Whether or not the first and/or last OBU is fragmented is signaled in the
+aggregation header. Fragmentation may occur regardless of how the X flag
+is set.
+
+The AV1 specification allows OBUs to have an optional size field called 
+obu_size (also leb128 encoded), signaled by the obu_has_size_field flag 
+in the OBU header. To minimize overhead, the obu_has_size_field flag SHOULD 
+be set to zero in all OBUs except fragmented OBUs where it MUST be set to
```
Comment (bjorn-terelius) [inline av1-rtp-spec.md]: Why signal the OBU size (in the bitstream) for fragmented OBUs? The RTP payload format lets us infer the size of each fragment, and the Z,Y flags helps us find the first and last fragment. The parser should be able to assemble all  fragments and just sum their sizes.

File: av1-rtp-spec.md (hunk 1 above)
Comment (aboba) [reply av1-rtp-spec.md]: Fixed.

File: av1-rtp-spec.md (hunk 2 above)
Comment (aboba) [reply av1-rtp-spec.md]: Fixed.

**Comment by aboba** [comment]
As discussed in the June 3 meeting, PR should be updated to provide the formula for calculation of the length of the last OBU.

**Comment by aboba** [comment]
Updated the length calculation per Bjorn's suggestion.

**Pull request merged**